magefiles/magefile.go

```diff
@@ -447,11 +447,76 @@ func (ci CI) setRequiredEnvVars() error {
 				envVarPrefix = "SPI_OPERATOR"
 				imageTagSuffix = "spi-image"
 				testSuiteLabel = "spi-suite"
-
 				// spi also requires service-provider-integration-oauth image
 				im := strings.Split(os.Getenv("CI_SPI_OAUTH_IMAGE"), "@")
 				os.Setenv("SPI_OAUTH_IMAGE_REPO", im[0])
 				os.Setenv("SPI_OAUTH_IMAGE_TAG", fmt.Sprintf("redhat-appstudio-%s", "spi-oauth-image"))
+			case strings.Contains(jobName, "multi-platform-controller"):
+				envVarPrefix = "MULTI_PLATFORM_CONTROLLER"
+				imageTagSuffix = "multi-platform-controller"
+				testSuiteLabel = "multi-platform-controller"
+				klog.Infof("going to create new Tekton bundle remote-build for the purpose of testing multi-platform-controller PR")
+				var err error
+				var defaultBundleRef string
+				var tektonObj runtime.Object
+
+				tag := fmt.Sprintf("%d-%s", time.Now().Unix(), util.GenerateRandomString(4))
+				quayOrg := utils.GetEnv(constants.DEFAULT_QUAY_ORG_ENV, constants.DefaultQuayOrg)
+				newJavaBuilderPipelineImg := strings.ReplaceAll(constants.DefaultImagePushRepo, constants.DefaultQuayOrg, quayOrg)
```

Comment:
would it be better to name this variable to something like `newBuildPipelineImg`

Reply:
copy/paste error